xml/cl.xml

```diff
@@ -5590,9 +5597,16 @@ server's OpenCL/api-docs repository.
             <require>
                 <type name="CL/cl.h"/>
             </require>
-            <require comment="Interop tokens">
+            <require>
+                <type name="cl_context_memory_initialize_khr"/>
+            </require>
+            <require comment="cl_context_properties">
                 <enum name="CL_CONTEXT_MEMORY_INITIALIZE_KHR"/>
             </require>
+            <require comment="cl_device_terminate_capability_khr">
```

Comment:
Should probably be cl_context_memory_initialize_khr instead of cl_device_terminate_capability_khr?

Reply:
Yes, thank you - too much copy-paste!